mmcls/apis/hub.py

```diff
@@ -0,0 +1,171 @@
+# Copyright (c) OpenMMLab. All rights reserved.
+import copy
+import fnmatch
+import os.path as osp
+from os import PathLike
+from pathlib import Path
+from typing import List, Union
+
+from mmengine.config import Config
+from mmengine.utils import get_installed_path
+from modelindex.load_model_index import load
+from modelindex.models.Model import Model
+
+from .inference import init_model
+
+
+def _expand_config_path(metainfo: Model,
+                        config_prefix: Union[str, PathLike] = None):
+    if config_prefix is None:
+        config_prefix = osp.dirname(metainfo.filepath)
+
+    if metainfo.config is None or osp.isabs(metainfo.config):
+        config_path: str = metainfo.config
+    else:
+        config_path = osp.abspath(osp.join(config_prefix, metainfo.config))
+
+    return config_path
+
+
+class ModelHub:
```

Comment:
How about using `ModelZoo` to keep consistent with documentation?

Reply:
I think users are more familiar with `hub`, which also aligns with the `mmengine.hub` package.